docs/api.md

```diff
@@ -1,24 +1,25 @@
-# auro-alerts
+# auro-alert
 
 ## Attributes
 
-| Attribute | Type      | Description                   |
-|-----------|-----------|-------------------------------|
-| `fixed`   | `Boolean` | uses px values instead of rem |
+| Attribute     | Type      | Description                        |
+|---------------|-----------|------------------------------------|
+| `error`       | `Boolean` | Turns alert into error style       |
+| `fixed`       | `Boolean` | uses px values instead of rem      |
+| `information` | `Boolean` | Turns alert into information style |
+| `success`     | `Boolean` | Turns alert into success style     |
+| `warning`     | `Boolean` | Turns alert into warning style     |
```

Comment:
The only attr that should be here is `fixed`. All other boolean attrs should be removed from the API docs.